notifications_utils/recipient_validation/phone_number.py

```diff
@@ -233,16 +232,35 @@ def _validate_forced_international_number(phone_number: str) -> phonenumbers.Pho
 
         return None
 
+    @property
+    def prefix(self):
+        """
+        Returns the international dialing code for looking up data in our international_billing_rates.yml file
+
+        in our billing rates yml file, countries in the North American Numbering Plan (+1) may fall under
+        US/Canada/Dominican Republic (just +1) or they may have their own specific area code within the plan, eg
+        Montserrat with numbers like "+1 664 xxx xxxx". This means we need to check the area code first to see if
+        it's a regular area code or a full country code.
+        """
+        if self.number.country_code == 1:
+            country_and_area_code = phonenumbers.format_number(self.number, phonenumbers.PhoneNumberFormat.E164)[1:5]
+            if country_and_area_code in INTERNATIONAL_BILLING_RATES:
+                return country_and_area_code
+        return str(self.number.country_code)
+
     def is_uk_phone_number(self):
         """
-        Returns if the number's country code is +44 - note, this includes any regions that also use the +44 country code
-        such as isle of man/jersey/guernsey. You can distinguish these with `number._is_a_crown_dependency_number`
+        Returns if the number starts with +44. Note, this includes international numbers for crown dependencies such as
+        jersey/guernsey.
+
+        # TODO: check if we still need this - looking at api, this might be able to be removed entirely since it's
```

Comment:
We might want to look into this - also the function name is confusing - it points at boolean return but it doesn't really return a boolean.

Reply:
we decided to leave it be for now - tests use it, and it's not a priority to get rid of it.